src/SFA.DAS.EmployerFinance/Data/TransactionRepository.cs

```diff
@@ -257,5 +257,17 @@ public async Task<TransactionDownloadLine[]> GetAllTransactionDetailsForAccountB
 
             return transactionDownloadLines.OrderByDescending(txn => txn.DateCreated).ToArray();
         }
+
+        public async Task<decimal> GetAllPaymentsForLastYear(long accountId)
+        {
+            var parameters = new DynamicParameters();
+            parameters.Add("@AccountId",accountId,DbType.Int64);
+
+            return await _db.Value.Database.Connection.ExecuteScalarAsync<decimal>(
+                sql: "[employer_financial].[GetAllPaymentsForLastYearByAccountId]",
```

Comment:
As above commend, SPROC should probably be renamed as it is returning a sum of the payment amounts not the payment lines themselves

Reply:
Renamed to match